ui/screens.py

```diff
@@ -461,8 +461,7 @@ def __init__(self, app):
         optsLyt.addWidget(self.spinner, 0, 1, Q.ALIGN_RIGHT)
 
         # Open staking window. Button is initally hidden until sync is complete.
-        self.stakeBttn = btn = app.getButton(SMALL, "Staking")
-        btn.setVisible(False)
+        btn = app.getButton(SMALL, "Staking")
```

Comment:
No need to hide the staking button anymore with the changes in this PR. This and the gap address synchronization handling should make the sync much faster.